src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> FRUITS= new HashMap<>();
```
i think this should be uppercase, because it is already defined as constant(static final)

Reply:
I think that we can`t call this map as constant, for now i think it`s better to not use uppercase